aroma/plotting.py

```diff
@@ -23,15 +23,23 @@ def classification_plot(in_file, out_dir):
     out_dir : str
         Output directory.
     """
+    try:
+        package = 'seaborn'
+        __import__(package)
+    except ImportError:
+        LGR.warning('Seaborn is needed for plotting, isntalling')
+        pip.main(['install', package])
```

Comment:
Is there a cleaner, more pythonic way of dealing with this @tsalo ?

Reply:
I Think this is the cleanest way. Actually, if you don't wanna install anything I would put a return after the warning message so the pipeline finishes correctly even if there is no plotting

Reply:
I would not raise an exeception since there is the data denoising step after this which won't be executed if we completely stop the pipeline. A warning is better in my opinion

Reply:
Okay, that sounds good!